logic/path/src/selectors/dfs.rs

```diff
@@ -241,6 +241,7 @@ where
         destination: Address,
         min_hops: usize,
         max_hops: usize,
+        blacklist: Vec<CorruptedChannelEntry>,
```

Comment:
This would work faster if it was something like `HashSet<SrcDstPair>` where:
```rust
#[derive(Clone,Copy,Debug,PartialEq,Eq, Hash)]
pub struct SrcDstPair(Address,Address);
```